test/format-support.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-env mocha */
+'use strict'
+
+const chai = require('chai')
+const dirtyChai = require('dirty-chai')
+const expect = chai.expect
+chai.use(dirtyChai)
+const BlockService = require('ipfs-block-service')
+const dagCBOR = require('ipld-dag-cbor')
+
+const IPLDResolver = require('../src')
+
+module.exports = (repo) => {
```

Comment:
Would it make sense to add a  test that a format is not dynamically loaded when it was already statically added?

Reply:
Done!